src/api/spec/features/webui/requests/diff_comments_spec.rb

```diff
@@ -35,8 +39,50 @@
 
       it 'displays the changes after adding a new comment' do
         expect(page).to have_text('This is a new reply')
-        expect(page).to have_text('target_project/target_package > package_a.changes')
+        expect(page).to have_text('target_project/package_a > package_a.changes')
+      end
+    end
+  end
+
+  describe 'create diff comment' do
+    before do
+      skip('This scenario is not posible under mobile') if mobile?
```

Comment:
💭  It would be nice if you would explain why...

Reply:
Done